packages/roosterjs-editor-plugins/lib/pluginUtils/DragAndDropHelper.ts

```diff
@@ -24,29 +35,50 @@ export default class DragAndDropHelper<TContext, TInitValue> implements Disposab
         private context: TContext,
         private onSubmit: (context: TContext, trigger: HTMLElement) => void,
         private handler: DragAndDropHandler<TContext, TInitValue>,
-        private zoomScale: number
+        private zoomScale: number,
+        isMobile: boolean = false
     ) {
-        trigger.addEventListener('mousedown', this.onMouseDown);
+        this.eventNames = DragAndDropHelper.getMouseEventNames(isMobile);
+        trigger.addEventListener(this.eventNames.mousedown, this.onMouseDown);
+    }
+
+    /**
+     * Generate event names based on different platforms
+     */
+    static getMouseEventNames(isMobile: boolean): MouseEventNames {
+        if (isMobile) {
+            return {
+                mousedown: 'touchstart',
+                mousemove: 'touchmove',
+                mouseup: 'touchend',
+            }
+        } else {
+            return {
+                mousedown: 'mousedown',
+                mousemove: 'mousemove',
+                mouseup: 'mouseup',
+            }
+        }
     }
```

Comment:
Just put this as a global constant, since it can never be changed.

const EventNames = Browser.isMobile ?  // You need to add this check in Browser.ts
 { ... } : {...}

Reply:
Refactored.